viewer-prototype/src/browser/trace-viewer/components/xy-output-component.tsx

```diff
@@ -70,13 +70,16 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
     renderTree(): React.ReactNode {
         this.onSeriesChecked = this.onSeriesChecked.bind(this);
         this.onCollapse = this.onCollapse.bind(this);
-        return <XYTree
-            entries={this.state.XYTree}
-            collapsedNodes={this.state.collapsedNodes}
-            checkedSeries={this.state.checkedSeries}
-            onChecked={this.onSeriesChecked}
-            onCollapse={this.onCollapse}
-        />;
+        return this.state.XYTree.length 
```

Comment:
Is this change needed for this patch here?

Reply:
yes because I'm using the parent tree data to initialize filteredNodes in tree. If no data is sent, the result of filter shows No entries found at the beginning

Reply:
This method though now returns undefined, so you should add it to the return type